Danki2/Assets/Scripts/Actor/Actor.cs

```diff
@@ -10,14 +10,14 @@ public abstract class Actor : MonoBehaviour
     private EffectTracker _effectTracker;
 
     // Start is called before the first frame update
-    void Start()
+    public virtual void Start()
```

Comment:
I think all of these Unity functions can be private (or protected in this case), they get called anyway (I think reflection or something) so may as well not have them public